tests/backend/localmongodb/test_queries.py

```diff
@@ -58,7 +58,7 @@ def test_write_assets():
     cursor = conn.db.assets.find({}, projection={'_id': False})\
                            .sort('id', pymongo.ASCENDING)
 
-    assert cursor.count() == 3
+    assert cursor.collection.count_documents({}) == 3
```

Comment:
Is there a specific reason for this change? CI seems to build fine regardless

Reply:
I know, but if you see here, there is a tremendous amount of noise after the tests pass. This is just resolving those deprecation warnings.

Reply:
e.g. here: https://travis-ci.org/bigchaindb/bigchaindb/jobs/428168863